python_bindings/readme.md

```diff
@@ -57,18 +58,15 @@ The best way to get set up is to use a virtual environment:
 ```console
 $ python3 -m venv venv
 $ . venv/bin/activate
-$ pip install -r requirements.txt 
+$ python3 -m pip install -U setuptools wheel
```

Comment:
Drive-by change, I presume, but what is this about?

Reply:
Regarding `pip` versus `python3 -m pip`, this prevents issues with overwriting open files on Windows while upgrading, see here: https://stackoverflow.com/a/60786507/2137996